packages/shared/src/components/squads/SquadMemberItemAdditionalContent.tsx

```diff
@@ -81,7 +81,7 @@ function SquadMemberItemAdditionalContent({
     return (
       <>
         <SquadMemberBadge
-          className={!isLoggedUser && 'my-auto mr-2'}
+          className={isLoggedUser ? 'mr-10' : 'mr-2'}
```

Comment:
Do we not still need the my-auto for vertical alignment?

Reply:
From what I see it does nothing.

Reply:
Ah it has flex on parent indeed. :)